content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -0,0 +1,228 @@
+to deploy only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines as part of your local environment for test purposes using the supplied kustomize manifests.
+
+You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and [kustomize](https://kustomize.io/).
+
+{{% alert title="Installation options for Kubeflow Pipelines standalone" color="info" %}}
+This guide currently describes how to install Kubeflow Pipelines standalone
+on Kind,K3s and K3s on WSL2 other configurations will be added here covering minikube(TBD) and others.
+{{% /alert %}}
+
+## Before you get started
+
+Working with Kubeflow Pipelines Standalone requires a Kubernetes cluster as well as an installation of kubectl.
+
+### Download and install kubectl
+
+Download and install kubectl by following the [kubectl installation guide](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+You need kubectl version 1.14 or higher for native support of kustomize.
+
+### Set up your cluster on Kind
+
+kind is a tool for running local Kubernetes clusters using Docker container “nodes”.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+See the guide [to install and configure kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+For simplicity we report below the basic steps here.
+First download and move kind executable somewhere in your path directory:
+On Linux:
+
+```
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/<KIND-VERSION>/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /some-dir-in-your-PATH/kind
+```
+On Mac:
+
+```
+ brew install kind
+```
+On Windows from within and administrative PowerShell console:
+
+```
+curl.exe -Lo kind-windows-amd64.exe https://kind.sigs.k8s.io/dl/<KIND-VERSION>/kind-windows-amd64
+Move-Item .\kind-windows-amd64.exe c:\some-dir-in-your-PATH\kind.exe
+```
+On Windows via Chocolatey [https://chocolatey.org/packages/kind](https://chocolatey.org/packages/kind):
+
+```
+choco install kind
+```
+Note, kind use containerd as default container-runtime hence you cannot use the standard kubeflow pipeline manifests.
+
+**References**:
+
+  * [Kind - Quick Start Guide](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+  * [Kind - Known Issues](https://kind.sigs.k8s.io/docs/user/known-issues/)
+
+  * [Kind - Working Offline](https://kind.sigs.k8s.io/docs/user/working-offline/)
+
+### Create your cluster on Kind
+
+Creating a Kubernetes cluster is as simple as ```kind create cluster```.
+
+This will bootstrap a Kubernetes cluster using a pre-built node image - you can find it on docker hub kindest/node. If you desire to build the node image yourself see the building image section. To specify another image use the --image flag.
+
+By default, the cluster will be given the name kind. Use the --name flag to assign the cluster a different context name.
+
+### Set up your cluster on K3S
```

Comment:
_(I'm sharing my refactored section on `k3s` here, in case you want to use it.)_

_(Feel free to change this.)_

_(Because I can't have "nested" code blocks in Markdown (three back ticks) without breaking the preview here, add them before and after command lines, if necessary.)_

```
## Setting up a cluster on k3s

k3s is a fully compliant Kubernetes distribution with the following enhancements:

* Packaged as a single binary.
* Lightweight storage backend based on sqlite3 as the default storage mechanism. etcd3, 
MySQL, Postgres also still available.
* Wrapped in simple launcher that handles a lot of the complexity of TLS and options.
* Secure by default with reasonable defaults for lightweight environments.
* Simple but powerful “batteries-included” features have been added, such as: a local storage 
provider, a service load balancer, a Helm controller, and the Traefik ingress controller.
* Operation of all Kubernetes control plane components is encapsulated in a single binary and 
process. This allows k3s to automate and manage complex cluster operations like distributing 
certificates.
* External dependencies have been minimized (just a modern kernel and cgroup mounts needed). 
k3s packages required dependencies, including:

  * containerd
  * Flannel
  * CoreDNS
  * CNI
  * Host utilities (iptables, socat, etc)
  * Ingress controller (traefik)
  * Embedded service loadbalancer
  * Embedded network policy controller

You can find the the official k3s installation script to install it as a service on systemd- or openrc-based 
systems on the official [k3s website](https://get.k3s.io). 

To install K3s using that method, run the following command:

```
```
curl -sfL https://get.k3s.io | sh -
```

```

**Note:** You cannot use the standard Kubeflow Pipeline manifests as kind uses 
[containerd](https://github.com/containerd/containerd) as a default container-runtime.

**References**:

* [K3S: Quick Start Guide](https://rancher.com/docs/k3s/latest/en/quick-start/)

* [k3s: Known Issues](https://rancher.com/docs/k3s/latest/en/known-issues/)

* [k3s: FAQ](https://rancher.com/docs/k3s/latest/en/faq/)

### Creating a cluster on k3s

1. To create a Kubernetes cluster on K3s, use the following command:

```
```
sudo k3s server &`
```

```
This will bootstrap a Kubernetes cluster `kubeconfig` written to `/etc/rancher/k3s/k3s.yaml`.

```
```
2. [EXPLAIN THIS COMMAND]:

```
```
sudo k3s kubectl get node
```

```
3. Run the below command on a different node. `NODE_TOKEN` comes from `/var/lib/rancher/k3s/server/node-token` 
on your server

```
```
sudo k3s agent --server https://myserver:6443 --token ${NODE_TOKEN}
```

```